File: SDKs/amazonaws.com/clouddirectory/2017-01-11/typescript/src/sdk/models/shared/batchlistattachedindicesresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { IndexAttachment } from "./indexattachment";



// BatchListAttachedIndicesResponse
/** 
 * Represents the output of a <a>ListAttachedIndices</a> response operation.
**/
export class BatchListAttachedIndicesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=IndexAttachments", elemType: IndexAttachment })
  indexAttachments?: IndexAttachment[];

  @SpeakeasyMetadata({ data: "json, name=NextToken" })
  nextToken?: string;
}

File: SDKs/amazonaws.com/clouddirectory/2017-01-11/go/pkg/models/shared/batchlistattachedindicesresponse.go
package shared

// BatchListAttachedIndicesResponse
// Represents the output of a <a>ListAttachedIndices</a> response operation.
type BatchListAttachedIndicesResponse struct {
	IndexAttachments []IndexAttachment `json:"IndexAttachments,omitempty"`
	NextToken        *string           `json:"NextToken,omitempty"`
}
